packages/model-viewer/src/features/scene-graph/texture-info.ts

```diff
@@ -56,7 +56,7 @@ export class TextureInfo extends ThreeDOMElement implements
   // Holds a reference to the glTF file data.
   [$gltf]: GLTF;
   // Holds a reference to the Three data that backs the material object.
-  [$material]: MeshStandardMaterial|null;
+  [$material]: Set<MeshStandardMaterial>|null;
```

Comment:
Does this need `|null`? Should we just initialize it to an empty set?

Reply:
Also, should the name be plural: `$materials`?

Reply:
Unfortunately it does need the null, this is to support the ad-hoc creation of a Texture. The Texture needs a TextureInfo at construction however in the ad-hoc case there is no material that the texture is being applied to.

Rename, done.